### PR TITLE
api/v1.0/events POST - Create an Event

### PR DESCRIPTION
**Pull request by fileames**
- Event model for database created.
- Minor path updates.
- api/v1.0/events POST created

### CODE REVIEW

File: practice-app/website/api/doc/events_POST.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        location:
+          type: string
+          description: Address of the event, given using basic English.
+          example: Trabzon Akyazı Stadium
+        creator_user:
+          type: integer
+          description: Id of the user creating the event. Id must be registered to a user.
+          example: 12434
+responses:
+  200:
```
Comment (berkatil) [inline practice-app/website/api/doc/events_POST.yml:32]: This should be 201

Comment (fileames) [reply practice-app/website/api/doc/events_POST.yml:32]: You are right

File: practice-app/website/templates/create_event.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            type="text"
+            class="form-control"
+            id="name"
+            name="name"
+            placeholder="Enter Event Name"
+            required
+          />
+        </div>
+
+        <div class="form-group">
```
Comment (38programmer61) [inline practice-app/website/templates/create_event.html:20]: Date should be required.

Comment (fileames) [reply practice-app/website/templates/create_event.html:20]: Corrected

**Pull request merged**